criu/crtools.c

```diff
@@ -170,6 +170,7 @@ int main(int argc, char *argv[], char *envp[])
 		goto usage;
 	}
 
+	util_init();
```

Comment:
pls write a comment why it should be initialized here and update the commit message with more details.

Reply:
The following commit is a good example: https://github.com/checkpoint-restore/criu/commit/e8087fcff6feae2384c00b0145bf13e168785cc2

Reply:
@rst0git This is a bad example. It has an empty commit message and zero comments;).

Reply:
Ok I'll update that one :) Thanks!

Reply:
Done in 3c4e1cb73ea72a984ddac917a19dd40dddf6fa55